### PR TITLE
Debug UI basics

### PR DESCRIPTION
**Pull request by keeganhauser**
Added text on screen for the FPS and added a very basic keyboard overlay. Since there's no player yet, I'll wait on finishing up adding all the debug info we wanted. Feel free to flame me for poor variable names or comments.

### CODE REVIEW

File: TKGame/TKGame.cs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ﻿using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
 using Microsoft.Xna.Framework.Input;
+using Myra;
```
Comment (NathanGr33n) [inline TKGame/TKGame.cs:4]: Might want to include a comment to describe what Myra is for those who haven't seen it before

File: TKGame/Debug.cs [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+// Myra is a library that allows us to add GUI components.
+// https://github.com/rds1983/Myra
+using Myra.Graphics2D.UI;
+using Microsoft.Xna.Framework.Input;
+
+namespace TKGame
+{
+    public class GameDebug
+    {
+        public static bool DebugMode { private get; set; }
```
Comment (NathanGr33n) [inline TKGame/Debug.cs:16]: You can add summary comments to variables and properties. Might be useful for others who may need to use your variables in the future.

**Review by NathanGr33n — APPROVED** [review]
Looks good to me. I have never used Myra before, so I commented it might be useful to add some more comments throughout explaining some of the Myra aspects of the code.

File: TKGame/TKGame.cs (hunk 1 above)
Comment (keeganhauser) [reply TKGame/TKGame.cs:4]: Fair, I have a small comment in Debug.cs, but I can move it to the main file

File: TKGame/Debug.cs (hunk 2 above)
Comment (keeganhauser) [reply TKGame/Debug.cs:16]: Also fair, I'll see what I can add that'll help and push it up

**Pull request merged**